src/domUpdates.js

```diff
@@ -0,0 +1,204 @@
+const domUpdates = {
+  createUserDisplay(user) {
+  let firstName = user.name.split(" ")[0];
+    let welcomeMsg = `
+      <div class="welcome-msg">
+        <h1>Welcome ${firstName}!</h1>
+      </div>`;
+    document.querySelector(".banner-image").insertAdjacentHTML("afterbegin",
+      welcomeMsg);
+  },
+
+  addToDom(recipeInfo, shortRecipeName, element) {
+  let cardHtml = `
+    <div class="recipe-card" id=${recipeInfo.id}>
+      <h3 maxlength="40">${shortRecipeName}</h3>
+      <div class="card-photo-container">
+        <img src=${recipeInfo.image} class="card-photo-preview" alt="${recipeInfo.name} recipe" title="${recipeInfo.name} recipe">
+        <div class="text">
+          <div>Click for Instructions</div>
+        </div>
+      </div>
+      <h4>${recipeInfo.tags[0]}</h4>
+      <img src="../images/apple-logo-outline.png" alt="unfilled apple icon" class="card-apple-icon">
+    </div>`
+   element.insertAdjacentHTML("beforeend", cardHtml);
+  },
```

Comment:
Love it.
I do wonder if we could come up with a better naming convention for "element" - - are we referring to the specific DOM meal card? Maybe we could later on just do a quick rename, but only if you both feel is necessary.